x-pack/docs/en/security/authorization/privileges.asciidoc

```diff
@@ -303,6 +303,10 @@ All {Ilm} operations relating to managing the execution of policies of an index
 or data stream. This includes operations such as retrying policies and removing
 a policy from an index or data stream.
 
+`manage_dlm`::
```

Comment:
I'd remove this whole section for now. See my other [comment](https://github.com/elastic/elasticsearch/pull/95512#discussion_r1206397157).